public_html/wp-content/plugins/wcpt/wcpt-meetup/class-meetup-admin.php

```diff
@@ -526,8 +526,8 @@ public static function notify_new_meetup_group_in_slack( $meetup ) {
 
 			$message = sprintf(
 				"Let's welcome the new WordPress meetup group%s%s, to the chapter! :tada: :community: :wordpress:\n%s",
-				isset( $city ) ? " in $city," : '',
-				isset( $organizer_slack ) ? " organized by @$organizer_slack" : '',
+				! empty( $city ) ? " in $city," : '',
+				! empty( $organizer_slack ) ? " organized by @$organizer_slack" : '',
```

Comment:
I think it's a bit easier to read like this:

`empty( $city ) ? '' : " in $city,"`

That way you don't have to mentally apply the `!` and it all flows smoother.